apps/web/src/app/[locale]/tracks/_components/track-challenge-card.tsx

```diff
@@ -4,11 +4,15 @@ import { Check } from '@repo/ui/icons';
 import clsx from 'clsx';
 import { useIsMobile } from '~/utils/useIsMobile';
 import type { Tracks } from './track-grid';
+import type { FC } from 'react';
```

Comment:
remove

Reply:
Done , but I'm not able to pass the prettier check :(